File: databinding-base/src/main/java/com/hangox/databinding/recycleview/HBRecyclerAdapter.java
package com.hangox.databinding.recycleview;

import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * Created With Android Studio
 * User hangox
 * Date 15/12/28
 * Time 下午3:16
 */
public abstract class HBRecyclerAdapter<VH extends RecyclerView.ViewHolder>
        extends RecyclerView.Adapter<VH> implements HBViewHolder.OnViewHolderClickListener{
    @Override
    public void onRootViewClick(View rootView, int index) {

    }
}

File: databinding-base/src/main/java/com/hangox/starter/recycleview/HBRecyclerAdapter.kt
package com.hangox.starter.recycleview

import android.view.View

import androidx.recyclerview.widget.RecyclerView

/**
 * Created With Android Studio
 * User hangox
 * Date 15/12/28
 * Time 下午3:16
 */
abstract class HBRecyclerAdapter<VH : RecyclerView.ViewHolder> : RecyclerView.Adapter<VH>(),
    HBViewHolder.OnViewHolderClickListener {
    override fun onRootViewClick(rootView: View, index: Int) {

    }
}
